UnitsNet.Tests/CustomCode/MassTests.cs

```diff
@@ -57,6 +57,12 @@ public class MassTests : MassTestsBase
 
         protected override double ShortHundredweightInOneKilogram => 0.022046226218487758;
 
+        protected override double EarthMassesInOneKilogram => 1.0 / 5.9722E+24;
```

Comment:
Same here, use values looked up in an online unit conversion calculator or some other reference source.